src/Component/Footer/TodosFilter.js

```diff
@@ -0,0 +1,114 @@
+import React, { useState, useEffect } from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import '../../styles/todo-list.css';
+import '../../styles/index.css';
+import '../../styles/filters.css';
+
+const handleFilter = (event, callFilter, criteria) => {
+  event.preventDefault();
+  callFilter(criteria);
+};
```

Comment:
you don't need event and preventDefault here
move this func to App, it should just save the filter value to the App's state